### ISSUE TITLE
XblDevAccount.exe signin doesn't support mail domains with .games ending

### ISSUE THREAD

**Issue by Dominga** [microsoft/xbox-live-developer-tools#81]
XblDevAccount.exe signin --name xxxxxxxx@yyyy.games
Error: signin failed
Response status code does not indicate success: 500 (Internal Server Error).

**Comment by rakhad** [comment]
Is this email associated with a Microsoft partner center account?

**Comment by Dominga** [comment]
> Is this email associated with a Microsoft partner center account?
Yes, this email is associated with a Microsoft partner center account and all my colleagues have the same issues  our emails have the same pattern xxxxx.yyyy@1cog.games

**Comment by rakhad** [comment]
Do you have access to the Xbox developer forums? Can you post this issue there so I can gather some personal information?

**Comment by Dominga** [comment]
> Do you have access to the Xbox developer forums? Can you post this issue there so I can gather some personal information?

Yes, I have. I posted question to Xbox developer forums
